src/sentry/static/sentry/app/utils/withOrganizations.tsx

```diff
@@ -23,7 +23,7 @@ const withOrganizations = <P extends InjectedOrganizationsProps>(
     State
   >({
     displayName: `withOrganizations(${getDisplayName(WrappedComponent)})`,
-    mixins: [Reflux.connect(OrganizationsStore, 'organizations')],
+    mixins: [Reflux.connect(OrganizationsStore, 'organizations') as any],
```

Comment:
Why do these need to cast to any?

Reply:
The reflux typings aren't very good, and apparently `Reflux.connect` [returns `void`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/reflux/v0/index.d.ts#L55), which is not compatible with [`mixins`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L710).

Reply:
Yea they're not good. I feel like we can roll out our own definitions that are better.

Reply:
meh, I would rather just get rid of reflux at some point. I just wanted the typings for the interface

Reply:
Should we update the typings?

Reply:
Yeah, that would let us correct the typings upstream.

Reply:
+1 on not investing time on typing reflux

Reply:
What do I need to do to get a +1 here :)